yxi/axi-calyx/axi-combined-calyx.futil

```diff
@@ -496,7 +495,7 @@ component m_awwrite_channel(
       // this contains blocking logic previously in its own group
       group do_aw_transfer {
           //assert AWVALID
-          is_awvalid.in = !(is_awvalid.out & AWREADY) & !awvalid_was_high.out ? 1'b1;
+          is_awvalid.in = !awvalid_was_high.out ? 1'b1;
```

Comment:
The same thing as `is_arvalid` [here](https://github.com/calyxir/calyx/pull/1846/files#r1449400350)